Watchman.Engine/Alarms/Defaults.cs

```diff
@@ -496,6 +496,75 @@ public static class Defaults
             }
         };
 
+        public static IList<AlarmDefinition> Alb = new List<AlarmDefinition>
+        {
+            new AlarmDefinition
+            {
+                Name = "Alb5xxErrorsHigh",
```

Comment:
Could we drop the Alb prefix? as other don't have it for the primary resource type?

Reply:
Removed the `Alb` prefix.